lib/preprocessors/glimmer.js

```diff
@@ -27,6 +28,10 @@ const closingTemplateTagRegex = /`, {.*}\)]/;
 // the capture group would contain `on,noop`
 const getScopeTokenRegex = /scope: \(\) => \({(.*)}\) }\)/;
 
+// eslint rules that check for undefined identifiers
+// this are the only rules we want to check within a template
```

Comment:
I still would like to see an explanation for why this is the only rule we care about.

Reply:
there is here: https://github.com/ember-cli/eslint-plugin-ember/blob/208e6b6e8a017f84671891c42953913a5eb05e3f/lib/preprocessors/glimmer.js#L84.
. Should i add an explanation here too? Like
"Because otherwise eslint would show errors unrelated to the original code and apply autofixes like spacing, comna etc into the template"

Reply:
Any where we could clarify comments is appreciated. I will leave it to your and @NullVoxPopuli's judgment.

Reply:
things look good from over here